<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileManagerDlg</class>
 <widget class="QDialog" name="FileManagerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileManagerDlg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="title">
        <property name="text">
         <string>File Manager</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="btnMinimize">
          <property name="text">
           <string/>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btnClose">
          <property name="text">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnMaximize">
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QListView" name="listViewPages">
       <property name="minimumSize">
        <size>
         <width>152</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>152</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pageTable">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>15</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>15</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="loadWidget" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4,0,0,5">
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>145</width>
                  <height>19</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="labelLoad">
                <property name="text">
                 <string>icon</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelLoadText">
                <property name="text">
                 <string>loading...</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>183</width>
                  <height>19</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="searchWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>9</number>
          </property>
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="CommonComboBox" name="comboBoxDate">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>174</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>174</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditWord">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>174</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>174</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSearch">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CommonComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">commoncombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxDate</tabstop>
  <tabstop>lineEditWord</tabstop>
  <tabstop>btnSearch</tabstop>
  <tabstop>listViewPages</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>btnMaximize</tabstop>
  <tabstop>btnMinimize</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../inst.qrc"/>
 </resources>
 <connections/>
</ui>
